roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-controllers.yml.j2

```diff
@@ -68,6 +72,8 @@ spec:
             - name: DATASTORE_TYPE
               value: kubernetes
 {% else %}
+            - name: ENABLED_CONTROLLERS
+              value: policy,namespace,serviceaccount,workloadendpoint,node
```

Comment:
This looks to be `node` only on the link you gave, am I missing something ?

Reply:
Yeah, nice catch, updated it.

Reply:
`node` only on the calico_datastore is kdd, if the datastore is etcd, we need this, see https://github.com/projectcalico/calico/blob/05bdc34b298940af451748481cc03d85e428f13f/manifests/calico-etcd.yaml#L655

Reply:
Oh right I missed the conditionnal structure :+1: